python/src/synapse/ml/dl/DeepVisionClassifier.py

```diff
@@ -0,0 +1,181 @@
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See LICENSE in project root for information.
+
+from pyspark import keyword_only
+from pyspark.ml.param.shared import Param, Params
+from synapse.ml.dl.LightningEstimator import LightningEstimator
+from synapse.ml.dl.LitDeepVisionModel import LitDeepVisionModel
+
+
+class DeepVisionClassifier(LightningEstimator):
+
+    backbone = Param(
+        Params._dummy(), "backbone", "backbone of the deep vision classifier"
+    )
+
+    num_layers_to_fine_tune = Param(
+        Params._dummy(),
+        "num_layers_to_fine_tune",
+        "number of last layers to fine tune for the model, should be smaller or equal to 3",
+    )
+
+    num_classes = Param(Params._dummy(), "num_classes", "number of target classes")
+
+    loss_name = Param(
+        Params._dummy(),
+        "loss_name",
+        "string representation of torch.nn.functional loss function for the underlying pytorch_lightning model, e.g. binary_cross_entropy",
+    )
+
+    optimizer_name = Param(
+        Params._dummy(),
+        "optimizer_name",
+        "string representation of optimizer function for the underlying pytorch_lightning model",
+    )
+
+    pretrained = Param(
```

Comment:
i think we'll always have to used pretrained weights if we arent training full net

Reply:
So you mean we just remove this parameter and fix it to True internally and do not let users train the full net?